src/components/componentDiff/ComponentDiffTab.js

```diff
@@ -204,6 +213,113 @@ ComponentDiffTabLoading.propTypes = {
 }
 
 
+const ExcludeSelectionDialog = ({ handleClose, diffComponents, handleExclude, handleExportDiff }) => {
+  const [excludedComponents, setExcludedComponents] = useState([])
+  const [excludedResources, setExcludedResources] = useState([])
+
+  const handleToggleComponent = (componentName) => {
+    const isExcluded = excludedComponents.includes(componentName)
```

Comment:
Are component versions not considered?
There are cases where multiple component versions are within the transitive closure of a component, e.g. multiple autoscaler versions as each supported k8s version requires a corresponding autoscaler version